enginetest/engine_only_test.go

```diff
@@ -71,7 +71,7 @@ func TestVariableErrors(t *testing.T) {
 func TestWarnings(t *testing.T) {
 	harness := enginetest.NewDefaultMemoryHarness()
 	if harness.IsUsingServer() {
-		t.Skip("depends on Warnings() method call on context")
+		t.Skip("tracking issue: https://github.com/dolthub/dolt/issues/692 and the one mentioned inside this issue as well")
```

Comment:
I think you meant https://github.com/dolthub/dolt/issues/6921 (looks like 692 is a really old PR)

```suggestion
		t.Skip("tracking issue: https://github.com/dolthub/dolt/issues/6921 and the one mentioned inside this issue as well")
```